### DISCUSSION TITLE
Possible to upload static website from web ui?

### DISCUSSION THREAD

**Discussion by afzl-wtu** [ZoeyVid/NPMplus#371] [Q&A] [answered]
Like HestiaCP is it possible to upload website files directly through web ui. It becomes  difficult to access the server through sftp and then put files in folder on server. And then come back to web ui to make it work?

**Comment by Zoey2936** [comment]
I think I don't have time for this, since it would be very complicated from my view, but if someone creates a PR, I would merge it. Why don't you use git to mirror the site?

**Reply by afzl-wtu** [reply]
You mean I should create a repo of any website I am going to put under NPM to github first? And then use git clone command to pull it from github to a local directory under NPM? or I am wrong?
I do not want to use ssh every time to git pull. It should be handled by Web ui.

**Reply by afzl-wtu** [reply]
I do not know html and css and Javascript much. I am Flutter developer otherwise I try to create a PR

**Reply by Zoey2936** [reply, marked as answer]
> You mean I should create a repo of any website I am going to put under NPM to github first? And then use git clone command to pull it from github to a local directory under NPM? or I am wrong? I do not want to use ssh every time to git pull. It should be handled by Web ui.

that is why I use this https://github.com/ZoeyVid/git-updater/blob/develop/compose.yaml, it pulls automatically the latest changes from a git repo
